app/main/views/buyers.py

```diff
@@ -1028,6 +1030,29 @@ def delete_a_brief(framework_slug, lot_slug, brief_id):
     return redirect(url_for('.buyer_dashboard'))
 
 
+@main.route('/frameworks/<framework_slug>/requirements/<lot_slug>/<brief_id>/withdraw', methods=['POST'])
+def withdraw_a_brief(framework_slug, lot_slug, brief_id):
+    get_framework_and_lot(
+        framework_slug,
+        lot_slug,
+        data_api_client,
+        allowed_statuses=['live', 'expired'],
+        must_allow_brief=True
+    )
+    brief = data_api_client.get_brief(brief_id)["briefs"]
+
+    if not is_brief_correct(brief, framework_slug, lot_slug, current_user.id) or brief['status'] != 'live':
```

Comment:
Would it be good to add `allowed_statuses` to this method so that it can be consistently called like with the `get_framework_and_lot` method? Seems like it might be neater than a separate check.

Reply:
Agreed - now done (for the other views that have Brief status checks as well).

Reply:
Nice! 👍